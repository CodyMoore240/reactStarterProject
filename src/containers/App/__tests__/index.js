import React from 'react'
import { shallow } from 'enzyme'
import shallowWithStore from 'shallowWithStore'
import { createMockStore } from 'redux-test-utils'
import ConnectedApp, { App } from '..'


//props
let props = {}

//create the wrapper
let wrapper = shallow(<App {...props}/>)

beforeEach(() => {
  wrapper.setProps(props)
})

describe(`App`, () => {
  it(`renders correctly`, () => {
    expect(wrapper).toMatchSnapshot()
  })

  describe(`connected`, () => {
    it(`should render correctly`, () => {
      const store = createMockStore(props)
      const connectedComponent = shallowWithStore(<ConnectedApp />, store)
      expect(connectedComponent).toMatchSnapshot()
    })
  })
})