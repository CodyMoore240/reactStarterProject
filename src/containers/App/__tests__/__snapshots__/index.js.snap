// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App connected should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`App renders correctly 1`] = `<Base />`;
